README.rst

```diff
@@ -182,4 +182,84 @@ A separate playbook can be used to invoke this functionality:
           name: caktus.django-k8s
           tasks_from: aws_s3
 
-Run with: ``ansible-playbook deploy-s3.yml``.
+Run with: ``ansible-playbook deploy-s3.yaml``.
+
+
+Amazon IAM: Adding a limited AWS IAM user for CI deploys
+````````````````````````````````````````````````````````
+
+In order to be able to deploy to AWS from CI systems, you'll need to be able to
+authenticate as an IAM user that has the permissions to push to the AWS ECR (Docker
+registry), and possibly need to be able to read a secret from AWS Secrets Manager (the
+``.vault_pass`` value). This playbook can create that user for you with the proper
+permissions. You can configure this with the following variables (defaults shown):
+
+```yaml
+k8s_ci_username: myproject-ci-user
+k8s_ci_repository_arn: "" # format: arn:aws:ecr:<REGION>:<ACCOUNT_NUMBER>:repository/<REPO_NAME>
+k8s_ci_vault_password_arn: "" # format: arn:aws:secretsmanager:<REGION>:<ACCOUNT_NUMBER>:secret:<NAME_OF_SECRET>
+```
+
+Only ``k8s_ci_repository_arn`` is required. The REPO_NAME portion can be found
+[here](https://console.aws.amazon.com/ecr/repositories). The `k8s_ci_vault_password_arn`
```

Comment:
Moving to `.rst`, looks like the links here need to be updated to Markdown-style